File: chapter03/hello3omp.c
#include <stdio.h> 
int main(void) 
{ 
   char greetings[] = "Hello world!"; 
   int a;
   #pragma omp parallel
   for ( a = 1; a < 11; a = a + 1 )
   	{
	printf("%s\n", greetings); 
   	}
return 0;
}

File: chapter03/hello3omp.f90
program SharedHello

implicit none

	character(len=16) :: greetings 
	greetings = "Hello World!" 
	print *, "Before parallel section: ", greetings 

   !$omp parallel private(greetings)
   greetings = "Saluton mondo!"
   print *, "Inside parallel section: ", greetings
   !$omp end parallel

   print *, "After parallel section:  ", greetings

end program SharedHello
